src/components/Jumbotron/Jumbotron.css

```diff
@@ -1,8 +1,11 @@
-.main-picture {
+.main {
   background: url(../../assets/Open-Air-Gampel.jpg), center, no-repeat;
+}
+
+.big-picture {
```

Comment:
if you consider it as sharable styles you have to move it in the common file with styles

Reply:
done